### PR TITLE
added AUR to README

### PR DESCRIPTION
**Pull request by Teddy-Kun**
I have created an AUR package for all of us Arch users and added a link to it in the README

### CODE REVIEW

**Comment by Umio-Yasuno** [comment]
Thanks @Teddy-Kun 
However, `amdgpu_top` uses crossterm-backend since v.0.1.2 , so it dose not depend on `ncurses`.
And `amdgpu_top` depends on `libdrm-amdgpu` .

**Comment by Teddy-Kun** [comment]
Oh I did not know about the `ncurses` thing. I updated it.
As for the `libdrm-amdgpu`: Arch does not have a separate package for it, there are only `libdrm` and `lib32-libdrm`.

File: README.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ OPTIONS:
 ### Packages
  * [Releases](https://github.com/Umio-Yasuno/amdgpu_top/releases)
     * .deb, .AppImage
+ * [AUR](https://aur.archlinux.org/packages/amdgpu_top)
+    * made by Teddy-Kun
```
Comment (Umio-Yasuno) [inline README.md:56]: Sorry, do we need the name of the package producer in the README?  
The README.md file should contain the necessary information.

Comment (Teddy-Kun) [reply README.md:56]: I personally don't think so, but some people value knowing that it is not made by the owner

Comment (Umio-Yasuno) [reply README.md:56]: Assuming there are such people, I do not wish to make an appeal to them.

Comment (Teddy-Kun) [reply README.md]: Ok, its removed

**Comment by Umio-Yasuno** [comment]
Apply, thanks.

**Pull request merged**